shell/packages/sandstorm-ui-grainlist/grainlist.html

```diff
@@ -3,7 +3,10 @@
     <div class="non-usage-info">
     <h1>Grains</h1>
     <div class="search-row">
-      <img title="Search" class="search-icon" src="/search.svg"><input class="search-bar" type="text" placeholder="search" value="{{ searchText }}">
+      <label>
+        <span title="Search" class="search-icon"></span>
```

Comment:
Hmm, is this a legal way to label inputs? I wonder if instead the `<span>` should have "Search" as its text content, but with `font-size: 0` in the css...